lib/apache/ant-zipscanner/README

```diff
@@ -0,0 +1,17 @@
+To rebuild ant-zipscanner, follow those steps, in a temporary directory:
+
+    unzip $GWT_TOOLS/lib/apache/ant-1.6.5-src.zip
+
+    patch -p0 --binary -i $GWT_TOOLS/lib/apache/ant-zipscanner-1.6.5.patch
+    # the --binary argument is necessary due to CRLF line endings
```

Comment:
instead, you can pass `-aa` to unzip to convert to local line endings when the files are unzipped. it appears that the zip file included all source files as if they were binary, so they are not decoded with platform-specific line endings.

Reply:
Done.

I've left the patch with CRLF line endings so it would (hopefully) still work on Windows (I have no idea how `unzip` in something like MSys or Cygwin would behave, so being conservative here; it only outputs `(Stripping trailing CRs from patch; use --binary to disable.)` for each patched file) but we could also just change the line endings to LF and assume Linux, OSX or similar.